modify date format and add date to ES id

anansi49 opened the pull request.

anansi49 committed:
modify date format and add date to ES id

Neemys reviewed (changes requested):
Fix travis fail on maven build

anansi49 committed:
fix date id

Neemys committed:
Ensure scripts are not running before launching them

Edit feature playbook to add a task that kill any running script before
relaunching them.

Neemys committed:
Remove mistakenly added slash from last commit
